cmake/BLTPrivateMacros.cmake

```diff
@@ -214,12 +214,14 @@ macro(blt_setup_cuda_source_properties)
         message( FATAL_ERROR "Must provide TARGET_SOURCES to the 'blt_setup_cuda_source_properties' macro")
     endif()
 
-
+    set ( _non_cuda_extensions .f .F .f90 .F90 .c .h )
```

Comment:
I think that we should switch away from extension-based "non-cuda" stuff... take a look at #28. We could do something similar for C files also.

Reply:
Fair enough. Reverted.